packages/components/src/menu-group/style.scss

```diff
@@ -10,6 +9,10 @@
 	}
 }
 
+.components-menu-group:has(> div:empty) {
```

Comment:
Alternative idea: could the `MenuGroup` component count its children, and if they are undefined (or a zero-length array) it adds a `data-empty` attribute? It would avoid relying on internal DOM structure (like `> div`)

Reply:
Not possible unfortunately for us because of how Slot/Fill works.